core/src/test/scala/integration/kafka/zk/ZkMigrationIntegrationTest.scala

```diff
@@ -74,7 +74,8 @@ object ZkMigrationIntegrationTest {
       MetadataVersion.IBP_3_7_IV0,
       MetadataVersion.IBP_3_7_IV1,
       MetadataVersion.IBP_3_7_IV2,
-      MetadataVersion.IBP_3_7_IV3
+      MetadataVersion.IBP_3_7_IV4,
+      MetadataVersion.IBP_3_8_IV0
```

Comment:
Notng that we are removing MetadataVersion.IBP_3_7_IV3 because it is "burned" and will never be used.

Reply:
Maybe we should keep this here as per the other comments?  Not sure, since the test will take some time, and maybe we don't want to spend it.

Reply:
I'm against adding it here as it doesn't add anything. IV4 and IV2 are the important ones.